.nf-core.yml

```diff
@@ -2,13 +2,6 @@ lint:
   files_unchanged:
     - .github/CONTRIBUTING.md
     - assets/multiqc_config.yaml
-    - .github/workflows/linting.yml
     - lib/NfcoreTemplate.groovy
     - lib/NfcoreSchema.groovy
     - assets/sendmail_template.txt
```

Comment:
> lib/NfcoreTemplate.groovy
   lib/NfcoreSchema.groovy
   assets/sendmail_template.txt

I added the two groovy files to this section as I had some tests failing, and it looks like the file `sendmail_template.txt` was also added recently by you. But perhaps we don't need them in this list anymore? What do you think?

Reply:
Yup! I forgot to remove them :)